1.8/usage/jobs/examples.md

```diff
@@ -5,56 +5,94 @@ feature_maturity: preview
 menu_order: 20
 ---
 
-# A Simple Job
-
-```json
-{
-  "id": "my-job",
-  "description": "A job that sleeps",
-  "run": {
-    "cmd": "sleep 1000",
-    "cpus": 0.01,
-    "mem": 32,
-    "disk": 0
-  }
-}
-```
-
-# A Job with a Schedule
-
-```
-{
-  "id": "my-scheduled-job",
-  "description": "A job that sleeps on a schedule",
-  "run": {
-    "cmd": "sleep 20000",
-    "cpus": 0.01,
-    "mem": 32,
-    "disk": 0
-  },
-  "schedules": [
+<a name="create-job"></a>
+# Create a Simple Job
+
+This JSON creates a simple job with no schedule.
+
+1. Create a JSON file with the following contents. 
+    ```json
     {
-      "id": "sleep-nightly",
-      "enabled": true,
-      "cron": "20 0 * * *",
-      "concurrencyPolicy": "ALLOW"
+      "id": "my-job",
```

Comment:
perhaps it is obvious... however you will use the `id` a lot to manage a job (just like pods).  it might be work calling out that the `id` in the job json is the `job_id`